scripts/kvmd-bootconfig

```diff
@@ -82,6 +89,10 @@ if [ -n "$FIRSTBOOT$FIRST_BOOT" ]; then
 		unset disk part npart label
 	fi
 
+    ### create NFS_Primary directory for use with NFS MSD after reboot ###
```

Comment:
This is not related with the current PR, pelase remove this part

Reply:
ok